translate/spec/quickstart_spec.rb

```diff
@@ -26,7 +26,7 @@
       load File.expand_path("../quickstart.rb", __dir__)
     }.to output(
       "Text: Hello, world!\n" +
-      "Translation: Привет, мир!\n"
+      "Translation: Привет мир!\n"
```

Comment:
Might be worth just switching to this for future changes. 
```suggestion
      "Translation:"
```

Reply:
Done. Thanks!